components/playlist/media_detector_component_manager.cc

```diff
@@ -230,7 +230,12 @@ void MediaDetectorComponentManager::SetUseLocalScriptForTesting() {
   }
 
   function getOGTagImage() {
-    return document.querySelector('meta[property="og:image"]')?.content
+    const url = document.querySelector('meta[property="og:image"]')?.content;
+    if (url && !url.startsWith('http://' && !url.startsWith('https://'))) {
```

Comment:
i think the parentheses are off here? instead of 

`(url && !url.startsWith('http://' && !url.startsWith('https://')))`

should it be

`(url && !url.startsWith('http://') && !url.startsWith('https://'))`

maybe add a test case for it?

Reply:
Oh, thanks.

Reply:
Modified and added a test